pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2903,6 +2903,22 @@ let
         };
       };
 
+      techtheawesome.rust-yew = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "rust-yew";
+          publisher = "techtheawesome";
+          version = "0.2.2";
+          sha256 = "sha256-t9DYY1fqW7M5F1pbIUtnnodxMzIzURew4RXT78djWMI=";
+        };
+        meta = {
+          description = "Provides some html language features for yew html macro.";
```

Comment:
And description should be more information (like at least tell everyone this is a vscode extension) and should not end with a period, see CONTRIBUTING.md for guidelines.